src/net/yapbam/export/HtmlFormatWriter.java

```diff
@@ -49,6 +83,18 @@ public void addValue(String value) throws IOException {
 	@Override
 	public void addFooter() throws IOException {
 		this.writer.append("</table>");
+		if (!StringUtils.isBlank(startBalance) || !StringUtils.isBlank(endBalance)) {
```

Comment:
Maybe it's more "standard" to have the startBalance in the header (The statements send by my bank have this structure: StatementId and start balance at the beginning, end balance at the end).